pkg/noun/manage.c

```diff
@@ -901,7 +915,6 @@ u3m_leap(c3_w pad_w)
   */
   {
     u3R = rod_u;
-    u3m_ward();
     _pave_parts();
```

Comment:
So, if I'm reading this right, we're no longer calling u3m_ward at all on leap beacuse of its dependency on u3R which may be unintialized. Instead, we have a call to u3e_ward in _pave_north. But there's no ward call in _pave_south. Shouldn't there be one?

Reply:
This is about calling ward *before* initializing the inner road, not after. Otherwise, the stores in `_pave_road()` can hit the guard page without the road boundaries having been updated.